docs/tutorials/getting-started-with-flask.rst

```diff
@@ -0,0 +1,114 @@
+Getting started with Rockcraft for Flask
+----------------------------------------
+
+In this tutorial you will learn how to create a Flask application, build a rock
+with the Flask application and run it.
+
+Prerequisites
+-------------
+- snap enabled system (https://snapcraft.io)
+- LXD installed (https://linuxcontainers.org/lxd/getting-started-cli/)
```

Comment:
LXD is not longer under linuxcontainers.org

Reply:
Fixed